services/movie_session.py

```diff
@@ -42,3 +41,12 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    movie_session = get_movie_session_by_id(movie_session_id)
```

Comment:
As per the checklist, it is recommended to use `get_user_model()` instead of `User`. Although you have not used `User` here, it's important to keep this in mind in other parts of your code as well.